### DISCUSSION TITLE
BAUMINVEST REFORESTATION PROJECT (Gold Standard project 2913)

### DISCUSSION THREAD

**Discussion by hai-yr** [opentaps/open-climate-investing#35] [Offsets]
Project activities (from original doc): 
- restoring forest landscapes in Costa Rica with native tree species in mixed stands and teak
- managing these forests sustainably with the aim of producing high quality timber for national and
international markets
- mitigating global warming and climate change by means of long-term carbon sequestration in trees and
growing forests. 

The project areas in question were formerly covered by tropical forests until they were converted to use for cattle ranching and growing cash crops.

"The project area of the BaumInvest Reforestation Project consists of three separated reforestation sites
within a radius of approximately 60 km located in the remote Northern Zone of Costa Rica.

Project duration: Minimum 30 years (2007 – 2037), however, the aim of the project is to establish a close-to-nature forest in
the mid- and long-term being managed in a sustainable way. "

Gold Standard Afforestation/Reforestation specific standards: "The crediting period shall be a minimum of 30 years and
maximum 50 years. The Project Developer shall select the crediting period based on the characteristics of the project."

Financial structure: 
"The financing of the proposed project activity is based on the closed-end funds (CEF) of the BaumInvest GmbH
& Co KG and BaumInvest 2 GmbH & Co KG, which were both successfully placed with 100% equity capital
provided by a few founding investors and many private small investors. This capital stock covers all costs for
the establishment of the plantation and includes capital reserves to finance the forest management and
maintenance of the proposed project activity over the first rotation period until revenues from sales of timber
are being generated."

Monitoring: The project documents include a certification and verification report from TÜV SÜD South Asia Pvt. Ltd. from the Monitoring Period 1: 01/09/2007 to 28/02/2015 

Buffer pool: 20% of credits

Project documents: https://platform.sustain-cert.com/public-luf-project/2781
Marketplace: https://marketplace.goldstandard.org/collections/projects/products/bauminvest-bauminvest-reforestation-project
Relevant standard: https://globalgoals.goldstandard.org/203-ar-luf-activity-requirements/
Contact email: info@bauminvest.de

**Comment by hai-yr** [comment]
I guess the first question that comes up for me is what is the incentive for Bauminvest to continue to own the land after it has sold all of the credits? It sounds like there is some revenue from sustainable logging, but is that enough to make it make sense to continue to own/protect all of the rest of the forest?

**Reply by hai-yr** [reply]
<img width="883" alt="Screen Shot 2022-08-16 at 7 54 33 AM" src="https://user-images.githubusercontent.com/39865976/184911077-0e8188ce-c0ec-4209-8d6e-f0133ef03152.png">

**Reply by sichen1234** [reply]
This sounds good.  Though I wonder if they're taking a tax credit for converting to a conservation easement, which again changes the economics of the project and therefore its additionality.

We should read through the prospectus and other information they have for investing in their fund, not buying the carbon credits.  It's available on their website at https://bauminvest.de/en/capital-increase/

**Comment by sichen1234** [comment]
This is an interesting project and similar to the [NicaForest](https://marketplace.goldstandard.org/products/nicaforest-high-impact-reforestation-program) in some ways.  They are both commercial tree (teak) plantations, thought his one claims native tree species, whereas NicaForest is just commercial teak.

My first questions are:

1. How do they justify this as being additional?  It seems the plantation was started with investment from these two BaumInvest funds.  
2. How are they calculating the amount of credits being issued?  Are they measuring the amount of trees each year?
3. What does the verification report say?  Did they measure everything or just "selective samples"?
4. If the verification report is from 2015, when are the credits from?
5. How do they calculate "leakage"?  Is any land around the project is being turned into grazing or farming?
6. Is there any updated verification after 2015?

**Reply by hai-yr** [reply]
1. How do they justify this as being additional? It seems the plantation was started with investment from these two BaumInvest funds.
So there must be a missing document because Gold Standard requires a document around additionality in the original filing (and I'm not seeing it) but not in the performance certification: https://globalgoals.goldstandard.org/standards/PRE-GS4GG-AF/ar-requirements_v0-9.pdf
2.. How are they calculating the amount of credits being issued? Are they measuring the amount of trees each year?
From what I can tell hey are modeling the growth of the thickness of the trees over the crediting period and using the area of the forest that has new growth to come up with a number.
3. What does the verification report say? Did they measure everything or just "selective samples"?
See attachment for example of sample locations
<img width="699" alt="Screen Shot 2022-08-15 at 3 50 44 PM" src="https://user-images.githubusercontent.com/39865976/184734003-f5c9a785-7f4f-4c66-b893-9f363c757467.png">

"The following amounts parcels were remeasured during the onsite visit
 La Virgen I & II a total of 14 out of 278 sample plots = 5%
 Las Delicias a total of 8 out 57 sample plots = 14%
 San Rafael a total of 8 out of 65 sample plots = 12 %
For each sample plot, about 30% of the measurements (DBH and Height) were repeated (IRL 48). The variance of the original data and the measured data was tested according to the F‐value test in order to detect if both variances origin from the same population. The result of the assessment showed that no significant difference between the differences of the two separate measurements could be identified, thus no bias was identified."

4. If the verification report is from 2015, when are the credits from?
The credits that are still being retired are between 2013 and 2019. I believe the rule is to have a new certification every 5 years.
7. How do they calculate "leakage"? Is any land around the project is being turned into grazing or farming?
The leakage should be calculated with the method in the GS requirements where they estimate the amount emissions of displaced farmland/livestock and subtract that out in the first year. I'm not seeing these calculations because it appears that document is missing.
8. Is there any updated verification after 2015?
Not that I can see, but I can email Gold Standard asking for that doc and any update on verification of the project.

**Reply by hai-yr** [reply]
<img width="995" alt="Screen Shot 2022-08-15 at 4 24 05 PM" src="https://user-images.githubusercontent.com/39865976/184740280-4d48e37d-c807-41bc-adc1-362914ce8ae7.png">

**Comment by sichen1234** [comment]
OK, let's see what they say on the additionality and leakage.

Meanwhile - What do they measure during verification?  Are they verifying
the thickness of the trees versus their model?  The density of the trees in
different areas to determine how many trees exist?

How could they use 2015 verification to issue credits after 2015?  How do
they know the condition of the forest after the last verification?
-----
Si Chen
Open Source Strategies, Inc.

*Why open source and blockchain for carbon accounting?  Video
<https://www.youtube.com/watch?v=eNM7V8vQCg4> and Blog Post
<https://www.opensourcestrategies.com/2022/06/01/why-open-source-carbon-accounting/>*
